src/components/Pagination.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+
+class Pagination extends React.Component {
+    constructor() {
+        super();
+
+        this.state = {
+            currentPage: 1
+        }
+
+        this.handleClick = this.handleClick.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleClick(e) {
+        const totalPages = Math.ceil(this.props.totalFilms / this.props.filmsPerPage)
+
+        if (e.target.className.includes("btn__next") && this.state.currentPage < totalPages) {
```

Comment:
As before, I would suggest separate click handlers and not using setState callbacks